.github/workflows/deploy-hubs.yaml

```diff
@@ -214,10 +213,9 @@ jobs:
       failure_utoronto: "${{ env.failure_utoronto }}"
       failure_victor: "${{ env.failure_victor }}"
 
-    # Only run this job on pushes to the default branch and when the job output is not
-    # an empty list
```

Comment:
I removed some comments that were hard to maintain as they were very detailed, but the code described is quite clear by itself so i think it didnt hurt readability